### ISSUE TITLE
Empty Sky/Block Light Mask support

### ISSUE THREAD

**Issue by Saiv46** [PrismarineJS/prismarine-chunk#112]
- [ ] Add new properties: `emptySkyLightMask` and `emptyBlockLightMask`
- [ ] Use them on load (API non-breaking change)
- [ ] Automatically add bits to these properties and nullify empty light sections
- [ ] Automatically remove bits when filling values in empty light sections

**Comment by Karang** [comment]
That already exist https://github.com/PrismarineJS/prismarine-chunk/blob/master/src/pc/1.16/ChunkColumn.js#L14

**Comment by Saiv46** [comment]
But these properties aren't used in ChunkColumn itself (e.g. loading, modifying, dumping)

**Comment by Karang** [comment]
Yes they are https://github.com/PrismarineJS/prismarine-chunk/blob/master/src/pc/1.16/ChunkColumn.js#L270

**Comment by Saiv46** [comment]
Do you even look at your examples?

**Comment by rom1504** [comment]
there is a doc here https://github.com/PrismarineJS/prismarine-chunk#api

what about the example ?
